.travis.yml

```diff
@@ -125,12 +125,16 @@ jobs:
       name: Subcommands on Kubernetes
       script: make test-subcommand
 
-    # Run the unit, sanity, and markdown tests
+    # Run the unit, sanity, markdown and coverage tests
     - stage: test
       name: Unit, Sanity, and Markdown Tests
-      script: make test-sanity test-unit test-markdown
+      before_install:
+        - go get github.com/mattn/goveralls
+      script:
+        - make test-sanity test-unit test-markdown test-coverage
```

Comment:
Won't test-unit and test-coverage run the tests twice? Once without coverage and once with coverage? I think you just want to run test-coverage.

Reply:
Hi @jmrodri,

I understand your concern. However, note that, for example, we are not able to cover the proxy tests. So, not necessary they will be the same. I think that it is fast and still better duplicate the call but keep them isolated.

Reply:
Could you explain this more?

I'm wondering why we couldn't update `make test-unit` to be:
```Makefile
test-unit test/unit: ## Run the unit tests
	$(Q)go test -coverprofile=coverage.out -covermode=count -count=1 -short ./cmd/... ./pkg/... ./internal/...
```

This seems like it runs the proxy tests just fine.

Reply:
Solved you are right. 
I think may was the tags used.